package.json

```diff
@@ -27,7 +27,7 @@
   "license": "SEE LICENSE IN https://tc39.es/ecma262/#sec-copyright-and-software-license",
   "homepage": "https://tc39.es/ecma262/",
   "dependencies": {
-    "ecmarkup": "^9.8.0"
+    "ecmarkup": "^9.8.1"
```

Comment:
since this is a patch version, the changes in this PR shouldn't be needed to land the ecmarkup upgrade, right?

Reply:
The patch fixes a bug which this PR would exercise, so yes, it is necessary.

Reply:
The question was whether we could do the ecmarkup upgrade first. This patch bump we could do, but the upcoming ecmarkup bump will be major, and should be included in this PR. So no point in doing this patch bump first.